function/metadata.go

```diff
@@ -87,3 +85,16 @@ func NewMetadata(application application.Application, logger logger.Logger) (Met
 	logger.Debug("riff metadata: %s", metadata)
 	return metadata, true, nil
 }
+
+func fileExists(file string) (bool, error) {
```

Comment:
Is this included in your test coverage? If it's not you should write a test.

Reply:
yeah, it's being covered.